adapters/rhythmone/params_test.go

```diff
@@ -0,0 +1,52 @@
+package rhythmone
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+func TestValidParams(t *testing.T) {
+	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
+	if err != nil {
+		t.Fatalf("Failed to fetch the json-schemas. %v", err)
+	}
+
+	for _, validParam := range validParams {
+		if err := validator.Validate(openrtb_ext.BidderRhythmone, json.RawMessage(validParam)); err != nil {
+			t.Errorf("Schema rejected rhythmone params: %s", validParam)
+		}
+	}
+}
+
+func TestInvalidParams(t *testing.T) {
+	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
+	if err != nil {
+		t.Fatalf("Failed to fetch the json-schemas. %v", err)
+	}
+
+	for _, invalidParam := range invalidParams {
+		if err := validator.Validate(openrtb_ext.BidderRhythmone, json.RawMessage(invalidParam)); err == nil {
+			t.Errorf("Schema allowed unexpected params: %s", invalidParam)
+		}
+	}
+}
+
+var validParams = []string{
+	`{"placementId":"123", "zone":"12345", "path":"34567"}`,
+}
+
+var invalidParams = []string{
```

Comment:
Since you require all 3 paramaters, a test where you have the incorrect type for one parameter could fail because the other two parameters are missing rather than because you use the wrong type. You should add valid values for the other two parameters when you test for an invalid parameter value so you can be sure you are testing that you fail because of that rather than possibly failing only because you omitted the other required parameters.